oap-server/server-starter/src/main/resources/ui-initialized-templates/rocketmq/rocketmq-cluster.json

```diff
@@ -538,7 +538,8 @@
       "entity": "Service",
       "name": "RocketMQ-Cluster",
       "id": "RocketMQ-Cluster",
-      "isRoot": false
+      "isRoot": false,
+      "isDefault": true
```

Comment:
We need to add some metrics here to show up on the hierarchy topology. Can refer to https://github.com/apache/skywalking/blob/b775e7f547cb91fd06d45ad43765715da3d46f6e/oap-server/server-starter/src/main/resources/ui-initialized-templates/virtual_database/virtual-database-service.json#L200

And `virtual-mq-service.json` allso needs too.

Reply:
@peachisai you still missing hierarchy node metrics here.